docsrc/content/index.md

```diff
@@ -13,7 +13,12 @@ The Fraud Structured Streaming Proofshop is a targeted event to prove structured
 
 ### What is included?
 
-A Powerpoint presentation will walk the prospect through DSE Spark Structured Streaming and the use case.
+This asset includes the following:
```

Comment:
I thought I saw an instance type recommendation somewhere. Can you put it here so that it's visible during asset deployment?